Generator/Sources/NeedleFramework/Parsing/BaseVisitor.swift

```diff
@@ -51,6 +51,18 @@ class BaseVisitor: SyntaxVisitor {
                 info("\(currentEntityName) (\(propertyName): \(propertyType)) property is private/fileprivate, therefore inaccessible on DI graph.")
                 return nil
             } else {
+                // See if this property has a code block associated with it (this skips the protocol memebers)
+                if let block = pattern.children.first(where: { $0.is(CodeBlockSyntax.self) }), let fullBody = block.as(CodeBlockSyntax.self) {
+                    // Make sure that the code block is a one-liner
+                    if let statement = fullBody.statements.first, fullBody.statements.count == 1 {
+                        let trimmed = statement.description.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
I've added a `trimmed` convenience method for this. You can do `.description.trimmed` directly